tests/testBackground.py

```diff
@@ -52,32 +42,18 @@
 import lsst.afw.display.ds9 as ds9
 import lsst.pex.exceptions as pexExcept
 
-try:
-    type(display)
-except NameError:
-    display = False
+# Set to True to display things in ds9.
+display = False
```

Comment:
I'm not sure whether we should keep this boilerplate or not. Given that it turned up 6 years ago in ff0c8fc4d3fb8d3e18c90ccf5d5efac743619b4a without explanation and has no description indicating why it exists, I'm happy to let it disappear.